{"trial_id": "01", "hyperparameters": {"space": [{"class_name": "Int", "config": {"name": "n_hidden", "default": 2, "conditions": [], "min_value": 2, "max_value": 8, "step": 1, "sampling": "linear"}}, {"class_name": "Int", "config": {"name": "n_neurons", "default": null, "conditions": [], "min_value": 16, "max_value": 256, "step": 1, "sampling": "linear"}}, {"class_name": "Float", "config": {"name": "learning_rate", "default": 0.0001, "conditions": [], "min_value": 0.0001, "max_value": 0.01, "step": null, "sampling": "log"}}, {"class_name": "Choice", "config": {"name": "optimizer", "default": "sgd", "conditions": [], "values": ["sgd", "adam"], "ordered": false}}], "values": {"n_hidden": 8, "n_neurons": 100, "learning_rate": 0.0012482904754698163, "optimizer": "sgd"}}, "metrics": {"metrics": {}}, "score": null, "best_step": 0, "status": "FAILED", "message": "Traceback (most recent call last):\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 274, in _try_run_and_update_trial\n    self._run_and_update_trial(trial, *fit_args, **fit_kwargs)\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\base_tuner.py\", line 265, in _run_and_update_trial\n    tuner_utils.convert_to_metrics_dict(\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner_utils.py\", line 132, in convert_to_metrics_dict\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner_utils.py\", line 132, in <listcomp>\n    [convert_to_metrics_dict(elem, objective) for elem in results]\n     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner_utils.py\", line 145, in convert_to_metrics_dict\n    best_value, _ = _get_best_value_and_best_epoch_from_history(\n                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\tuner_utils.py\", line 116, in _get_best_value_and_best_epoch_from_history\n    objective_value = objective.get_value(metrics)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ureca\\miniconda3\\envs\\TAA-py311\\Lib\\site-packages\\keras_tuner\\src\\engine\\objective.py\", line 59, in get_value\n    return logs[self.name]\n           ~~~~^^^^^^^^^^^\nKeyError: 'val_mae'\n"}